cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -85,7 +86,6 @@ case class CommandLineTool private(
         pull <- requirement.dockerPull.orElse(requirement.dockerImageId)
       } yield ValueAsAnExpression(WomString(pull))
 
-      import mouse.option._
```

Comment:
ToL: curious why you wanted to make it less well contained?

Reply:
Yeah I might move it back. I think there was an intermediate step where I was using this in more than one spot, but that's clearly not the case anymore.